install/kubernetes/cilium/values.yaml

```diff
@@ -1007,6 +1007,21 @@ operator:
       maxUnavailable: 1
     type: RollingUpdate
 
+  ## Affinity
+  ##
+  affinity:
```

Comment:
can you help to document this new attribute in https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/README.md as well ?

Reply:
the rest LGTM, thanks for your help working on this.

Reply:
I will take care of that.

Reply:
@sayboras my pleasure and I just pushed the updated README.md. Let me know if this is good.